Solving Issue #768. please verify with author. (#829)
MicrosoftDocs/OfficeDocs-o365seccomp: SecurityCompliance/controlling-your-data-using-customer-key.md

Within 48 hours, the active mailboxes encrypted using this policy will become associated with the updated key. Use the steps in [Determine the DEP assigned to a mailbox](controlling-your-data-using-customer-key.md#DeterminemailboxDEP) to check the value for the DataEncryptionPolicyID property for the mailbox. The value for this property will change once the updated key has been applied. 
  
> [!IMPORTANT]

> > To manage your keys using **Microsoft Teams** the steps to enable it are still the same that the one listed here. Mainly because you create a DEP and then assign it to the mailboxes.

Reviewer: ```suggestion
> When managing your keys using **Microsoft Teams**, the procedure to enable the keys remain the same that the one listed here. This is mainly because you create a DEP and then assign it to the mailboxes.
```

> > When managing your keys using **Microsoft Teams**, the procedure to enable the keys remain the same that the one listed here. This is mainly because you create a DEP and then assign it to the mailboxes.

Reviewer 2: ```suggestion
> If you manage your keys using **Microsoft Teams**, the procedure to enable the keys remains the same as the one listed here. This is mainly because you create a DEP and then assign it to the mailboxes.
```
